src/runtime/pkg/resourcecontrol/cgroups_darwin.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2023 Apple Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package resourcecontrol
+
+import (
+	"fmt"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+type DarwinResourceController struct{}
+
+func RenameCgroupPath(path string) (string, error) {
+	return "", fmt.Errorf("RenameCgroupPath not supported on Darwin")
```

Comment:
Prefer `errors.New` if no formatting (also go idiom is to start errors lowercased)

Reply:
ack -- I'll keep non-idiomatic just based on it being the function name?